docs/src/components/footer.js

```diff
@@ -6,6 +6,7 @@ import { height } from 'styled-system'
 import { PanelProvider, Wrapper } from './theme'
 import dta from '../images/dta-wordmark-white.svg'
 import { External } from './link'
+import { NOTIFY_BASE_URL } from './core/env-vars'
```

Comment:
I think I'd prefer to just use the env var directly rather than import from a global file of env vars. So just

```js
const notifyBaseURL =
  process.env. NOTIFY_BASE_URL || 'https://notify.gov.au'
```

Alternatively, what you've done reduces the chance of there being a variation in the `|| 'https://notify.gov.au'` part, so I would maybe propose this as a different way of doing this.

New file `src/url.js` (or `src/core/url.js`—whatever is best):

```js
const base = process.env. NOTIFY_BASE_URL || 'https://notify.gov.au';

export const path = (pat: string) => `${base}${pat}`; // or could be smarter to make sure you don't end up with double slashes, etc.
```

Then use it like:

```jsx
import * as url from "../wherever/url";

<Link to={url.path('/support')}>
  Support
</Link>
```

Now you encapsulate the config and logic nicely. And also super easy to test this module.

Thoughts?

Reply:
@andrewoh531 I'm just reading over my suggestion again. In my example I used React Router links which just need a path e.g. `/foo/bar`. In our code we have cases where we need full URLs. So how about this instead:

New file `src/url.js` (or `src/core/url.js`—whatever is best):

```js
const base = process.env. NOTIFY_BASE_URL || 'https://notify.gov.au';

const url = pat => `${base}${pat}`;

export default url;
```

Then use it like:

```jsx
import url, { path } from "../wherever/url"; // imported path just so show that it can be there if we need it too—can't recall if we need it in docs yet

<Link to={url('/support')}>
  Support
</Link>
```